### ISSUE TITLE
`createStore` with `serialize: ignore` won't ignores when used with factory

### ISSUE THREAD

**Issue by domosedov** [effector/effector#863]
**What is the current behavior:**
```ts
const app = createDomain();
const $a = createStore(0, { sid: 'a', serialize: 'ignore', domain: app });

const q = createOperation({
  name: 'factory',
  zodOutputContract: z.unknown(),
  effect: attach({
    source: $a,
    effect: (a) => a * 10,
  }),
});

(async () => {
  const scope = fork(app, {
    values: [[$a, 100]],
  });

  await allSettled(q.run, { scope });

  console.log(serialize(scope));
  // output
  //{
  //  "a": 100, <== should be omitted
  //  "fgv8oy|supabaseOperation:factory:status": "done",
  //  "fgv8oy|supabaseOperation:factory:data": 1000
  //}
})();
```

**Please provide the steps to reproduce and if possible a minimal demo of the problem via https://share.effector.dev, https://codesandbox.io or similar**

[stackblitz](https://stackblitz.com/edit/effector-vite-react-template-jsso6e?file=src%2Ffeatures%2Fmodel.ts,src%2Fapplication.tsx)

**What is the expected behavior:**

ignore stores with `serialize: 'ignore'` option

**Which versions of effector packages, and which browser and OS are affected by this issue? Did this work in previous versions of effector?:**
```
 "effector": "^22.5.2"
```

**Comment by domosedov** [comment]
minimal repo
https://share.effector.dev/FDFuu77n

**Comment by sergeysova** [comment]
@domosedov Repl doesn't support factories at all.

This is correct reproduce: https://stackblitz.com/edit/effector-vite-react-template-ijgn5l?file=src%2Fmain.tsx,src%2Ffactory.ts
> Look at the console

**Comment by zerobias** [comment]
Looks like its already fixed in 22.5.2 or 23 version
